rotary/rules/rotary.smk

```diff
@@ -518,16 +538,14 @@ rule short_read_contamination_filter:
         "{sample}/benchmarks/qc/short/short_read_contamination_filter.benchmark.txt"
     params:
         contamination_filter_kmer_length = config.get("contamination_filter_kmer_length"),
-        all_contaminant_references = ','.join(CONTAMINANT_REFERENCE_GENOMES_NCBI +
```

Comment:
@jmtsuji If we need commas I think there is a ways to access input variables inside params. https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#non-file-parameters-for-rules
Maybe a lambda function?

Reply:
Interesting -- this would be an elegant solution. Can you try this out?

Reply:
Implemented.